app/components/team_sidebar/add_team/add_team.tsx

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
This file was formerly the `.index.tsx` in the `add_team` directory.  The only change is to observe the `currentUserId` to pass to `add_team_slide_up.tsx`  Another option is to:
 
* keep `add_team.tsx` as the `index.tsx` 
* create subcomponent `add_team_slide_up/` with `.index.tsx` to get the current user id